charts/apiclarity/templates/deployment.yaml

```diff
@@ -65,6 +65,10 @@ spec:
                   key: postgresql-password
             - name: STATE_BACKUP_FILE_NAME
               value: /apiclarity/state.gob
+         {{- range $key, $val := .Values.env.plugins }}
```

Comment:
where `env.plugins` is defined, didn't saw changes in the values file

Reply:
you are right... I was sure I added them